hr_attendance_reason/models/hr_employee_base.py

```diff
@@ -10,3 +10,6 @@ class HrEmployeeBase(models.AbstractModel):
     show_reasons_on_attendance_screen = fields.Boolean(
         related="company_id.show_reasons_on_attendance_screen", store=True
     )
+    required_reason_on_attendance_screen = fields.Boolean(
+        related="company_id.required_reason_on_attendance_screen", readonly=False
```

Comment:
It doesn't make sense to be `readonly=False`, isn't it?
```suggestion
        related="company_id.required_reason_on_attendance_screen"
```